lighthouse-core/runner.js

```diff
@@ -131,6 +137,7 @@ class Runner {
           benchmarkIndex: artifacts.BenchmarkIndex,
         },
         lighthouseVersion,
+        axeVersion,
```

Comment:
does this belong in `environment` instead of the toplevel? I guess it might depend on our resolution to runtimesettings v. footer?

Reply:
> does this belong in `environment` instead of the toplevel? I guess it might depend on our resolution to runtimesettings v. footer?

+1 to moving to `environment` regardless. Not even axe puts the axe version number at the top level of their results object :)

Reply:
+1. but how about a toplevel `credits` object though? or `environment.credits` ?